README.md

```diff
@@ -20,34 +20,15 @@ To use Authentication in a SwiftPM project:
 ]),
 ```
 
-3. Add `import Authentication` and `import UserDetails` in your source code.
+3. Add `import Authentication` in your source code.
 
 ## Package description
 
-The Authentication package authenticates a users details and enables them to log into their account securely. This is done by providing them with a login session and token.
-
-The package also integrates [openID](https://openid.net/developers/how-connect-works/) AppAuth and conforms to its standards, and also uses `NetworkClient` from the [Networking](https://github.com/govuk-one-login/mobile-ios-networking) package.
+The Authentication package implements OpenID Connect to authenticates a user with an remote server.
```

Comment:
Small typo for authenticates, I'll fix it

Reply:
As we're on grammar, should it also be `with a remote server` instead of `an remote server`?